addons/website_slides/models/slide_question.py

```diff
@@ -15,18 +15,22 @@ class SlideQuestion(models.Model):
     question = fields.Char("Question Name", required=True, translate=True)
     slide_id = fields.Many2one('slide.slide', string="Content", required=True, ondelete='cascade')
     answer_ids = fields.One2many('slide.answer', 'question_id', string="Answer", copy=True)
+    correct_answers = fields.Integer("Correct Answers", compute='_count_answers')
+    wrong_answers = fields.Integer("Wrong Answers", compute='_count_answers')
+    answers_validation_error = fields.Char("Error on Answers", compute='_count_answers')
     # statistics
     attempts_count = fields.Integer(compute='_compute_statistics', groups='website_slides.group_website_slides_officer')
     attempts_avg = fields.Float(compute="_compute_statistics", digits=(6, 2), groups='website_slides.group_website_slides_officer')
     done_count = fields.Integer(compute="_compute_statistics", groups='website_slides.group_website_slides_officer')
 
     @api.constrains('answer_ids')
     def _check_answers_integrity(self):
+        error_msg = ""
         for question in self:
-            if len(question.answer_ids.filtered(lambda answer: answer.is_correct)) != 1:
-                raise ValidationError(_('Question "%s" must have 1 correct answer', question.question))
-            if len(question.answer_ids) < 2:
-                raise ValidationError(_('Question "%s" must have 1 correct answer and at least 1 incorrect answer', question.question))
+            if question.correct_answers == 0 or question.wrong_answers == 0:
+                error_msg += f'- Question "{question.question}" from "{question.slide_id.name}"\n'
```

Comment:
To help avoiding errors in translation, we prefer to keep it as short as possible in the string. Something like
```suggestion
                error_msg += _('- Question "%(question_title)s" from "%(slide_name)"\n', question_title=question, slide_name=question.slide_id.name)               
```

But actually, we can avoid this complexity with f'{question.slide_id.name} : {question.question}'